pkg/storage/stores/local/boltdb_shipper_table_client_test.go

```diff
@@ -54,10 +54,6 @@ func TestBoltDBShipperTableClient(t *testing.T) {
 	err = tableClient.DeleteTable(context.Background(), "table1")
 	require.NoError(t, err)
 
-	// cortex does not omit empty directories from the list
-	// ToDo: change the code in cortex to remove empty directories from the list
-	ensureEmptyAndRemoveDirectory(t, path.Join(tempDir, storageKeyPrefix, "table1"))
```

Comment:
Cortex code is modified to handle this scenario. This test is not required any more
@sandeepsukhani Hope this is fine.